src/main/java/gregtech/common/tileentities/machines/multi/GT_MetaTileEntity_AssemblyLine.java

```diff
@@ -130,6 +133,9 @@ public boolean checkRecipe(ItemStack aStack) {
                 this.mMaxProgresstime /= 2;
             }
         }
+        if (this.mEUt > 0) {
+            this.mEUt = -this.mEUt;
```

Comment:
Pretty sure i did something about that shortly after adding the multiblock. Likely got overwritten in some later git change after that. Happend a few times.